.vscode/settings.json

```diff
@@ -40,5 +40,10 @@
         "yotta",
         "zepto",
         "zetta"
-    ]
+    ],
+    "editor.formatOnSave": false,
+    "javascript.format.enable": false,
+    "typescript.format.enable": false,
+    "json.format.enable": false,
```

Comment:
Most of these probably are better off in your user's settings than the project settings

Reply:
Actually on all other Plotly projects that I see (I've been in 5 so far, and it seems true for projects in the Dash land too), `.vscode` is added to `.gitignore`. And it makes sense. VSCode is awesome, but we can't assume that every developer will be using it, especially when we have an open source project. Here we seem to be using it mostly for cSpell, which can have its own config file AFAIKT.

We've also been using prettier as an auto-formatter everywhere (which is awesome BTW!).

Of course every developer that works on `dash-table` could just set the global config to whatever is above, and then disable auto-formatter for each project locally one by one. It's this project that's going against the grain though.

Reply:
Leaving this out possibly for another PR...